app/build.gradle

```diff
@@ -18,7 +18,7 @@ repositories {
 android {
     compileSdkVersion 24
     //noinspection GradleDependency
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```

Comment:
Please don't change the buildToolsVersion in this commit. :)
@kunall17 Do you think we should upgrade the buildToolsVersion?